site/src/patterns/chat.md

```diff
@@ -269,3 +269,28 @@ The following markup utilizes various components and atomics to create a chat ex
 	</article>
 </section>
 ```
+
```

Comment:
Should we also document `.tag-suggestion` on the [tag component page](https://design.learn.microsoft.com/components/tag.html)?